config/routes.rb

```diff
@@ -11,6 +11,9 @@
   devise_for :users,
     controllers: { omniauth_callbacks: "users/omniauth_callbacks" }
 
+  # NOTE: temporary page, remove when pages admin module is finished
+  get '/about', to: 'pages#about', as: :about
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.